engines/msat_ic3ia.cpp

```diff
@@ -0,0 +1,191 @@
+/*********************                                                        */
+/*! \file msat_ic3ia.h
+** \verbatim
+** Top contributors (to current version):
+**   Ahmed Irfan, Makai Mann
+** This file is part of the pono project.
+** Copyright (c) 2019 by the authors listed in the file AUTHORS
+** in the top-level source directory) and their institutional affiliations.
+** All rights reserved.  See the file LICENSE in the top-level source
+** directory for licensing information.\endverbatim
+**
+** \brief A backend using the open-source IC3IA implementation from
+**        Alberto Griggio. Available here:
+**        https://es-static.fbk.eu/people/griggio/ic3ia/index.html
+**
+**/
+
+#include "engines/msat_ic3ia.h"
+
+#include "assert.h"
+#include "smt-switch/msat_factory.h"
+#include "smt-switch/msat_solver.h"
+#include "utils/logger.h"
+
+using namespace ic3ia;
```

Comment:
perhaps, we shouldn't import namespace. It will make the code more readable -- we will know that we are calling ic3ia library methods